Create ui-c8y-1020-13-1-application-for-access-mapping-are-getting-from-application-6740-graft-release.md (#2310)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1020-13-1-application-for-access-mapping-are-getting-from-application-6740-graft-release.md

---
date: ""

> title: application for access mapping are getting from application… (#6740) [GRAFT][release/cd] (#6775)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Access mappings for applications are now retrieved from the application
```

> title: The list of applications that can be configured for SSO has been narrowed down to applications available to the tenant.

Reviewer: ```suggestion
title: List of applications available in SSO configuration now limited to applications available to the tenant
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: q3kclF6pO
    label: Authentication
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-60033
version: 1020.13.1
---

> application for access mapping are getting from application… (#6740) [GRAFT][release/cd] (#6775)

Author: The Claude AI suggested to use the following body:
```suggestion
In the past, access mappings for applications were not properly retrieved from the application itself, which could lead to incorrect access permissions. With this change, access mappings are now correctly fetched from the respective application. This ensures that the appropriate access rights are applied based on the application's configuration, providing a more reliable and secure access control mechanism for users.
```

> The list of applications that can be configured for access by users logging in through SSO has been narrowed down to only include applications available to the tenant.

Reviewer 2: The list of applications that can be chosen in Access mapping of SSO configuration has been limited to include only those available to the respective tenant.

Reviewer 3: Out of interest do we show the applications that are not available to the tenant but grey them out?  Or do we put an information icon to tell the user that if they cannot see the application they are looking for that the tenant does not have access to it?

Reviewer 2: As I know it simply does not show up in the drop down as explained in the ticket comments https://cumulocity.atlassian.net/browse/MTM-60033
